.github/workflows/ci.yml

```diff
@@ -231,10 +231,17 @@ jobs:
       - name: Test rego policy interpreter
         run: gotestsum --format standard-verbose --debug -- -mod=mod -gcflags=all=-d=checkptr ./internal/regopolicyinterpreter
 
+      - name: Run containerd-shim-runhcs-v1 tests
+        shell: powershell
+        run: |
+          powershell {
+            cd '../..'
+            go build -trimpath -o './test/containerd-shim-runhcs-v1' ./cmd/containerd-shim-runhcs-v1
```

Comment:
Why output this to ./test/containerd-shim-runhcs-v1?

Reply:
the shim tests want the executable in the same working directory